### DISCUSSION TITLE
Why was "feat: Add health checks for kubernetes-external-secrets" reverted?

### DISCUSSION THREAD

**Discussion by Aenima4six2** [argoproj/argo-cd#6306] [Q&A] [answered]
Looks like the release notes for `v2.0.2` indicated  `Revert "feat: Add health checks for kubernetes-external-secrets`, what was the reason for this ?

**Comment by alexmt** [comment, marked as answer]
Hello @Aenima4six2 , we had to revert the health check from v2.0 release because it was using standard libraries which are disabled for security reasons: https://github.com/argoproj/argo-cd/issues/6039#issuecomment-821585563

Both standard libraries [support](https://github.com/argoproj/argo-cd/issues/6050) and kubernetes-external-secrets health check will be available in v2.1

**Reply by Aenima4six2** [reply]
Thank you!
